website/www/site/content/en/documentation/sdks/python-machine-learning.md

```diff
@@ -47,9 +47,16 @@ For more infomation about machine learning with Apache Beam, see:
 * [About Beam ML](/documentation/ml/about-ml)
 * [RunInference notebooks](https://github.com/apache/beam/tree/master/examples/notebooks/beam-ml)
 
+## Support and limitations
+
+- The RunInference API is supported in Apache Beam 2.40.0 and later versions.
+- PyTorch and Scikit-learn frameworks are supported. Tensorflow models are supported through tfx-bsl.
+- The RunInference API supports batch and streaming pipelines.
+- The RunInference API supports local and remote inference.
```

Comment:
local to the runner worker

Reply:
I updated this, but I don't know what local to the runner worker means, so someone should make sure that what I wrote is correct, and maybe we should clarify.